cime_config/testdefs/testlist_clm.xml

```diff
@@ -1835,6 +1835,22 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
+  <test name="ERS_D_Ld15" grid="5x5_amazon" compset="I2000Clm50FatesRs" testmods="clm/FatesColdSeedDisp">
+    <machines>
+      <machine name="cheyenne" compiler="gnu" category="fates"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:40:00</option>
+    </options>
+  </test>
+  <test name="PEM_D_Ld15" grid="5x5_amazon" compset="I2000Clm50FatesRs" testmods="clm/FatesColdSeedDisp">
+    <machines>
+      <machine name="cheyenne" compiler="gnu" category="fates"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:40:00</option>
```

Comment:
Comment here as well.

Reply:
Should we also add a failing ERP test to aux_clm for this?

Reply:
Let's add the ERP test when the PEM test starts passing. You could add a comment about that.